otcextensions/sdk/dds/v3/_proxy.py

```diff
@@ -59,3 +60,74 @@ def flavors(self, region, engine_name):
             region=region,
             engine_name=engine_name
         )
+
+    # ======= Instance =======
+    def create_instance(self, **attrs):
+        """Create a new instance from attributes
+
+        :param dict attrs: Keyword arguments which will be used to create
+            a :class:`~otcextensions.sdk.dds.v3.instance.Instance`,
+            comprised of the properties on the Instance class.
+
+        :returns: The result of an instance creation.
+
+        :rtype: :class:`~otcextensions.sdk.dds.v3.instance.Instance`
+        """
+        return self._create(_instance.Instance, **attrs)
+
+    def delete_instance(self, instance, ignore_missing=True):
+        """Delete an instance
+
+        :param instance: The value can be either the ID of an instance or a
+            :class:`~otcextensions.sdk.dds.v3.instance.Instance` instance.
+        :param bool ignore_missing: When set to ``False``
+            :class:`~openstack.exceptions.ResourceNotFound` will be
+            raised when the instance does not exist.
+            When set to ``True``, no exception will be set when
+            attempting to delete a nonexistent instance.
+
+        :returns: ``None``
+        """
+        return self._delete(
+            _instance.Instance,
+            instance,
+            ignore_missing=ignore_missing,
+        )
+
+    def get_instance(self, **attrs):
+        """Get a single instance
+
+        :param instance: The value can be either the ID of an instance or a
+            :class:`~otcextensions.sdk.dds.v3.instance.Instance` instance.
+
+        :returns: One :class:`~otcextensions.sdk.dds.v3.instance.Instance`
+        """
+        return self._get(_instance.Instance, **attrs)
+
+    def find_instance(self, name_or_id, ignore_missing=True):
+        """Find a single instance
+
+        :param name_or_id: The name or ID of a instance.
+        :param bool ignore_missing: When set to ``False``
+            :class:`~openstack.exceptions.ResourceNotFound` will be
+            raised when the resource does not exist.
+            When set to ``True``, None will be returned when
+            attempting to find a nonexistent resource.
+
+        :returns:
+            One :class:`~otcextensions.sdk.dds.v3.instance.Instance` or None.
+        """
+        return self._find(_instance.Instance,
+                          name_or_id,
+                          ignore_missing=ignore_missing)
+
+    def instances(self, **params):
```

Comment:
mb you should rename it into `list_instances`?

Reply:
no, this is common for all resources